pkg/tsdb/cloudwatch/response_parser_test.go

```diff
@@ -0,0 +1,62 @@
+package cloudwatch
+
+import (
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudwatch"
+	"github.com/grafana/grafana/pkg/components/null"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestCloudWatchResponseParser(t *testing.T) {
+	Convey("TestCloudWatchResponseParser", t, func() {
```

Comment:
We're trying to avoid using Go convey moving forward. So for new tests I would suggest to convert this file to using the standard library. See https://github.com/grafana/grafana/blob/master/contribute/style-guides/backend.md#testing for more information.